frontend/src/metabase/css/core/colors.module.css

```diff
@@ -89,6 +89,8 @@
   --mb-color-background-inverse: var(--mb-color-bg-black);
   --mb-color-background-brand: var(--mb-base-color-brand-40);
   --mb-color-icon-primary: var(--mb-base-color-brand-40);
+  --mb-color-icon-disabled: var(--mb-base-color-gray-30);
+  --mb-color-icon-secondary: var(--mb-base-color-brand-20);
```

Comment:
I think these semantic colors make sense. And it works with the [base PR wonderfully.](https://github.com/metabase/metabase/pull/49356)

But I want to resolve [this discussion](https://github.com/metabase/metabase/pull/49356#discussion_r1825793447) even after we got this solution whether there should be cases where we could define base colors outside `colors.module.css`. The discussion should block this patch from going into the base PR though.

Reply:
for sure. could you put a "request changes" on this PR? Then we can block it until we figure that out

Reply:
I mistype. I was gonna type `shouldn't block`.

That discussion belongs to the main PR.

Reply:
hehehehe fair enough

Reply:
corrected it now 🙏